File: dataset/output-bk/OPENAI_gpt-4o/Task122_OPENAI_gpt-4o.java
import java.util.HashMap;
import java.util.Scanner;

public class Main {
    private static HashMap<String, String> passwords = new HashMap<>();

    public static void signup(String username, String password) {
        passwords.put(username, password);
        System.out.println("Signup successful!");
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter Username: ");
        String username = scanner.nextLine();
        System.out.println("Enter Password: ");
        String password = scanner.nextLine();
        signup(username, password);
        scanner.close();
    }
}

File: dataset/output-bk/OPENAI_gpt-4o/Task122_OPENAI_gpt-4o.cpp
#include <iostream>
#include <unordered_map>
#include <string>

std::unordered_map<std::string, std::string> passwords;

void signup(const std::string& username, const std::string& password) {
    passwords[username] = password;
    std::cout << "Signup successful!" << std::endl;
}

int main() {
    std::string username, password;
    std::cout << "Enter Username: ";
    std::cin >> username;
    std::cout << "Enter Password: ";
    std::cin >> password;
    signup(username, password);
    return 0;
}